incydr/cli/cmds/utils.py

```diff
@@ -66,7 +66,7 @@ def output_response_format(
 
 # WARNING: CSV output will not work for nested objects
 def output_models_format(
```

Comment:
Once everything is converted to this new logger we can remove these methods?

Reply:
yeah, I think at least for now we'll not implement a generic "handle all formats" function to allow for flexibility in each command. There will be a bit of duplication of if-statements, but maybe once we get every command working individually (with checkpointing, etc), we can see if something more generic like this makes sense.